File: VersaoAjustesLocaweb/src/MP.Client/MP/ctrlTemplateDeEmail.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace MP.Client.MP
{
    public partial class ctrlTemplateDeEmail : System.Web.UI.UserControl
    {
        protected void Page_Load(object sender, EventArgs e)
        {

        }

        public string TextoDoTemplate
        {
            get { return txtTextoDoTemplate.Content; }
            set { txtTextoDoTemplate.Content = value; }
        }

        public void AdicionaTextoNoTemplate(string texto)
        {
            txtTextoDoTemplate.Content = txtTextoDoTemplate.Content + " [" + texto + "] ";
        }

        public void Inicializa()
        {
            txtTextoDoTemplate.Content = "";
        }
    }
}

File: trunk/src/WorkSpace/ctrlTemplateDeEmail.ascx.vb
Public Class ctrlTemplateDeEmail
    Inherits System.Web.UI.UserControl

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load

    End Sub


    Public Property TextoDoTemplate() As String
        Get
            Return txtTextoDoTemplate.Content
        End Get
        Set(value As String)
            txtTextoDoTemplate.Content = value
        End Set
    End Property


    Public Sub AdicionaTextoNoTemplate(texto As String)
        txtTextoDoTemplate.Content = txtTextoDoTemplate.Content & " [" & texto & "]"
    End Sub

    Public Sub Inicializa()
        txtTextoDoTemplate.Content = ""
    End Sub

End Class